recipes/simdjson/all/conanfile.py

```diff
@@ -32,8 +38,8 @@ def _build_subfolder(self):
     @property
     def _compilers_minimum_version(self):
         return {
-            "gcc": "7.4",
-            "Visual Studio": "15.7",
+            "gcc": "8",
+            "Visual Studio": "16",
```

Comment:
The author tells that gcc 7 is enough.
Isn't that right?
And clang 6 should work as well.

https://github.com/simdjson/simdjson#quick-start

Reply:
Oh, I just saw the comment about 7.4, is it possible to keep 7.4

Reply:
No, due to CI of CCI.

Reply:
Well if we revert the changes... than will pass for CCI and anyone using custom compiler versions like it was before

Reply:
I don't know if it's very common, there is no silver bullet here.